evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -345,6 +349,7 @@ def test_output_format_checked():
                 Predicted Value: setosa
                 Target Value: setosa
                 Cross Entropy: 0.15
+                Index ID: {{index_0}}
```

Comment:
why does this have two curly braces?

Reply:
We call `.format` once to format the string with the template for the multiclass shap table (that'll remove the outer curly braces) and then in the test we call `.format` to format the answer with the expected index (that'll remove the  inner curly braces).